.github/workflows/ci.yml

```
@@ -19,6 +19,12 @@ jobs:
       with:
         go-version: '1.22'
         
+    - name: Clean Go module cache
+      run: rm -rf ~/go/pkg/mod || true
+          
+    - name: Cache Go modules
+      uses: actions/cache@v3
```

actionlint (1.7.9)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)